apps/web2/src/components/Menu.tsx

```diff
@@ -0,0 +1,100 @@
+import { Link } from "react-router-dom";
+import { Accordion, AccordionDetails, AccordionSummary, Box, Paper, Stack, Typography, styled } from "@mui/material";
+import { ExpandMore as ExpandMoreIcon } from "@mui/icons-material";
+import { FacebookLogo, InstagramLogo } from "../../../../packages/ui/src";
+
+interface LiItemProps {
+  label: string
+}
+
+export const Label = styled(Link)`
+  text-decoration: none;
+`;
+
+function LiItem({ label }: LiItemProps) {
+  return (
+    <Typography
+      variant="h3"
+      sx={(theme) => ({ color: theme.palette.text.primary, fontSize: 18, fontWeight: 500 })}
```

Comment:
https://mui.com/system/getting-started/the-sx-prop/#theme-aware-properties

El color que sea theme aware para evitar tener un callback

Reply:
changed to theme aware propertieres